Userland/Libraries/LibWeb/DOMTreeModel.cpp

```diff
@@ -130,26 +132,34 @@ GUI::Variant DOMTreeModel::data(const GUI::ModelIndex& index, GUI::ModelRole rol
     }
 
     if (role == GUI::ModelRole::Icon) {
-        if (type == "document")
+        if (type == "document"sv)
             return m_document_icon;
-        if (type == "element")
+        if (type == "element"sv) {
+            if (node_name == "SCRIPT"sv)
+                return m_script_icon;
+            if (node_name == "STYLE"sv)
+                return m_style_icon;
+            if (node_name == "LINK"sv)
+                if (node.has("attributes"sv) && node.get("attributes"sv).as_object().get("rel"sv).as_string_or("unknown") == "stylesheet"sv)
```

Comment:
Let's use an empty StringView for the or.

Reply:
Done 👍🏻